docs/src/APIs/Soil.md

```diff
@@ -17,6 +17,31 @@ ClimaLand.Soil.RichardsParameters
 ClimaLand.Soil.EnergyHydrologyParameters
 ```
 
+## Soil Constructor Methods
+
+```@docs
+ClimaLand.Soil.RichardsParameters(;
+    hydrology_cm::C,
+    ν::F,
+    K_sat::F,
+    S_s::F,
+    θ_r::F,
+) where {F <: Union{<:AbstractFloat, ClimaCore.Fields.Field}, C}
+ClimaLand.Soil.EnergyHydrologyParameters(
```

Comment:
Could we also add
https://github.com/CliMA/ClimaLand.jl/blob/32e96955b837ff4ec609f99553212abd19f9371a/src/standalone/Soil/Soil.jl#L242

https://github.com/CliMA/ClimaLand.jl/blob/32e96955b837ff4ec609f99553212abd19f9371a/src/standalone/Soil/Soil.jl#L299